### PR TITLE
chrome-cli 1.6.0 - added brave support

### PR DESCRIPTION
**Pull request by kxalex**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/chrome-cli.rb [hunk 1]
```diff
@@ -37,7 +37,14 @@ def install
...
     bin.install "build/Release/chrome-cli" => "chromium-cli"
-  end
+
+    # Brave builds; see:
+    # https://github.com/prasmussen/chrome-cli/issues/57
+    rm_rf "build"
+    inreplace "chrome-cli/App.m", "org.Chromium.chromium", "com.brave.Browser"
+    xcodebuild "SDKROOT=", "SYMROOT=build"
+    bin.install "build/Release/chrome-cli" => "brave-cli"
+ end
```
Comment (SMillerDev) [inline Formula/chrome-cli.rb:47]: This needs to be enabled upstream, we're not going to randomly patch browser names into some source.

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/chrome-cli.rb (hunk 1 above)
Comment (kxalex) [reply Formula/chrome-cli.rb]: What do you mean **enabled upstream**? 

Just followed the same pattern as for Canary (see line 30) and Chromium (see line 37) as the app does not support and the `chrome-cli` maintainer is not doing any fixes on it.

Comment (SMillerDev) [reply Formula/chrome-cli.rb]: There's a big difference between "patching in support for a beta release of the same code" and "patching in support for another browser" IMO. If this is something the author is fine with they can use a buildsystem that allows for this configuration. If they are not, we should not do this.

Comment (kxalex) [reply Formula/chrome-cli.rb]: Fair enough. I will close PR then, hopefully, the author will add such support but I feel he abandoned the project years ago.

**Pull request closed without merging**